plugins/catalog-unprocessed-entities/src/components/FailedEntities.tsx

```diff
@@ -27,12 +27,17 @@ import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import IconButton from '@material-ui/core/IconButton';
 import { Theme, makeStyles } from '@material-ui/core/styles';
-import { alertApiRef, useApi } from '@backstage/core-plugin-api';
+
+import { alertApiRef, errorApiRef, useApi } from '@backstage/core-plugin-api';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+import { catalogTranslationRef } from '@backstage/plugin-catalog/alpha';
```

Comment:
You wouldn't use this here but create your own, more details here: https://backstage.io/docs/plugins/internationalization. 

I'd say skip the Internationalization for now just to keep this PR focused on the feature your adding 👍